interpreter/cling/include/cling/Interpreter/Interpreter.h

```diff
@@ -667,6 +679,9 @@ namespace cling {
                         IgnoreFilesFunc_t ignoreFiles =
                           [](const clang::PresumedLoc&) { return false;}) const;
 
+    void CodeComplete(const std::string& line, size_t& cursor,
```

Comment:
Differs only by Capitalization - that's evil. Maybe `CodeCompleteInternal`? Do we even need a second function, or can we combine this with the first?